price-feeder/oracle/provider/huobi.go

```diff
@@ -384,6 +397,27 @@ func (p *HuobiProvider) setSubscribedPairs(cps ...types.CurrencyPair) {
 	}
 }
 
+// GetAvailablePairs return all available pairs symbol to susbscribe.
```

Comment:
Same suggestion as the others: 
```suggestion
// GetAvailablePairs returns all pairs to which the provider can subscribe.
```